bloom-housing/bloom
# feat: custom exception filter

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/3608)


## Description
This creates a simple starting custom exception handler. This will eventually act as a way to handle errors that we don't want making it to the front end

## How Can This Be Tested/Reviewed?
There really isn't a way to test this. You could create an endpoint locally that throws an error and make sure that error finds its way to the front end, but for now this acting just as a pass through

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
